src/package/SlateEditor/SlateEditor.js

```diff
@@ -15,7 +15,11 @@ class SlateEditor extends Component {
   }
 
   onChange (state) {
-    this.setState({ state })
+    const { onChange } = this.props;
+    this.setState({ state });
+    if(onChange) {
```

Comment:
@gabrielrtakeda Why no semi-colons? I write a lot of JavaScript and I've never heard of semi-colons not being used...?